### PR TITLE
Add example notebook showcasing KerasWrapper

### PR DESCRIPTION
**Pull request by huiwengoh**
Add example notebook showcasing KerasWrapper corresponding to the blogpost in https://github.com/cleanlab/web/pull/148

### CODE REVIEW

File: transformers_sklearn/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This notebook currently requires the latest developer version of cleanlab: pip install git+https://github.com/cleanlab/cleanlab.git
+ipywidgets==8.0.2
+numpy>=1.22.0
+pandas==1.5.0
+scikit_learn==1.0.2
```
Comment (jwmueller) [inline transformers_sklearn/requirements.txt]: are these 3 really needed or is the versions installed with cleanlab already ok?

File: transformers_sklearn/transformers_sklearn.ipynb [hunk 2]
```diff
@@ -0,0 +1,754 @@
...
+    "\n",
+    "To demonstrate how we can use this KerasWrapperModel, we will train a classifier (using a pretrained bert model) to classify positive and negative reviews on an Amazon Reviews dataset following the steps below:\n",
+    "\n",
+    "- Tokenize out text data using a tokenizer from a pretrained bert model \n",
+    "- Define our neural network model and wrap it using cleanlab's `KerasWrapperModel`\n",
+    "- Conduct a grid search to find the best parameters for the model using sklearn's `GridSearchCV`\n",
+    "- Using the optimal parameters found, train the classifier to classify the reviews\n",
+    "- To further improve the model performace, use cleanlab's `CleanLearning` to identify potential label issues in the dataset and retrain the classifier to get improved predictions"
```
Comment (jwmueller) [inline transformers_sklearn/transformers_sklearn.ipynb]:
```suggestion
    "- To further improve the model performance, use cleanlab's `CleanLearning` to identify potential label issues in the dataset, and retrain the classifier to get improved predictions"
```

File: transformers_sklearn/transformers_sklearn.ipynb [hunk 3]
```diff
@@ -0,0 +1,754 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56d44364",
+   "metadata": {},
+   "source": [
+    "Please install the dependencies specified in this [requirements.txt](https://github.com/cleanlab/examples/blob/master/transformers_sklearn/requirements.txt) file before running the notebook.  "
```
Comment (jwmueller) [inline transformers_sklearn/transformers_sklearn.ipynb]: also clarify this notebook should ideally be run with GPU for better efficiency?

File: transformers_sklearn/requirements.txt (hunk 1 above)
Comment (huiwengoh) [reply transformers_sklearn/requirements.txt]: versions installed with cleanlab are alright, these are just specified as versions used to run the current example - can remove if we think are not necessary to list

Comment (jwmueller) [reply transformers_sklearn/requirements.txt]: ya lets remove them, this'll presumably be stable with all cleanlab versions (and their dependencies) going forward

**Review by jwmueller — APPROVED** [review]
LGTM!

**Pull request merged**